core/templates/dev/head/pages/exploration_editor/exploration_editor.html

```diff
@@ -73,26 +73,26 @@
       </div>
       <!-- This is an ng-if, so that the preview loads the latest version of the exploration each time the tab is accessed. -->
       <div ng-if="getActiveTabName() === 'preview'">
-        {% include 'pages/exploration_editor/preview_tab/preview_tab.html' %}
+        <preview-tab></preview-tab>
       </div>
 
       <div ng-show="getActiveTabName() === 'stats'">
-        {% include 'pages/exploration_editor/statistics_tab/statistics_tab.html' %}
+        <statistics-tab></statistics-tab>
       </div>
 
       <improvements-tab ng-if="getActiveTabName() === 'improvements'">
       </improvements-tab>
 
       <div ng-show="getActiveTabName() === 'settings'">
-        {% include 'pages/exploration_editor/settings_tab/settings_tab.html' %}
+        <settings-tab></settings-tab>
```

Comment:
```diff
- <div ng-show="getActiveTabName() === 'TAB'">
-   <tab></tab>
- </div>
+ <tab ng-show="getActiveTabName() === 'TAB'"></tab>
```

Which style should we prefer?

Reply:
@brianrodri I am currently using the first one but if there are no regressions in the second style then I guess the second one would be fine!

@seanlip @kevinlee12 @DubeySandeep for their suggestions here.

Reply:
I have no strong opinion either way.

Reply:
My preference isn't strong either, I'd just like to see consistency.